src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/ReadResourcesValidation.feature.disabled

```diff
@@ -0,0 +1,56 @@
+# This is a rough draft feature for future use.
+@ignore
+Feature: Resources "Read" Operation validations
+
+        Background:
+            Given the Data Management Service must receive a token issued by "http://localhost"
+              And user is already authorized
+              And request made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "Sick Leave",
+                        "description": "Sick Leave",
+                        "effectiveBeginDate": "2024-05-14",
+                        "effectiveEndDate": "2024-05-14",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             Then it should respond with 201
+
+        @ignore
+        Scenario: Verify existing resources can be retrieved successfully
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors"
+             Then it should respond with 200
+
+        @ignore
```

Comment:
Can combine the next two into a single scenario.

Please replace the `16435704274945eeae6c6c45b281aad3` with `{id}`.

Reply:
Done